networking/bridge/sanity_check/metadata

```diff
@@ -0,0 +1,12 @@
+[General]
+name=networking/bridge/sanity_check
+owner=Fei Liu <feliu@redhat.com>
+description=Networking bridge sanity test
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=
```

Comment:
We could delete dependencies  in restraint entry if not used.
[restraint] entry is required  entry_point  minimally.

Reply:
Removed.